base/db/R/query.pft.R

```diff
@@ -83,19 +83,19 @@ query.pft_cultivars <- function(pft, modeltype = NULL, con) {
       suffix = c("", ".cvpft"))
     %>% dplyr::inner_join(
       dplyr::tbl(con, "cultivars"),
-      by = c("cultivar_id" = "id"),
+      by = c(".data$cultivar_id" = "id"),
       suffix = c("", ".cv"))
     %>% dplyr::inner_join(
-      dplyr::tbl(con, "species"),
-      by=c("specie_id" = "id"),
+      dplyr::tbl(con, ".data$species"),
+      by=c(".data$specie_id" = "id"),
```

Comment:
`.data` not needed inside strings.

```suggestion
      by=c("specie_id" = "id"),
```